.github/workflows/dev.yaml

```diff
@@ -15,40 +14,42 @@ jobs:
     environment:
       name: dev
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         name: checkout
 
-      - uses: hashicorp/setup-terraform@v1
-        with:
-          terraform_version: ${{ env.TF_VERSION }}
-          cli_config_credentials_token: ${{ secrets.TFC_TEAM_TOKEN }}
-
-      - name: Setup Infra
-        run: make apply
-
-      - uses: actions/cache@v2
+      - uses: actions/cache@v3
         name: Cache yarn
         with:
           path: ./.yarn/cache
-          key: ${{ runner.os }}-yarn-${{ hashFiles('./yarn.lock') }}
-          restore-keys: ${{ runner.os }}-yarn-${{ hashFiles('./yarn.lock') }}
+          key: pcc-backend
+          restore-keys: pcc-backend
 
       - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v1
+        uses: aws-actions/configure-aws-credentials@v1-node16
         with:
```

Comment:
note: node 16

Reply:
this iis for GHA errors, not related to our node version.